### PR TITLE
feature: add delete handler

### PR DESCRIPTION
**Pull request by sarmat86**
Support for del function from Vercel Blob
https://vercel.com/docs/storage/vercel-blob/using-blob-sdk#delete-a-blob

### CODE REVIEW

File: src/handlers/del.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+        const file = Bun.file(path.join(storePath, formattedUrl));
+        const isExists = await file.exists();
+
+        if (isExists) {
+          await unlink(path.join(storePath, formattedUrl));
+          await unlink(
+            path.join(storePath, formattedUrl + '._vercel_mock_meta_')
+          );
+          console.log(path.join(storePath, formattedUrl));
```
Comment (634750802) [inline src/handlers/del.ts]: add a `deleted` prefix or remove the `console.log`

File: src/handlers/del.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    const body = await request.json();
+
+    const urlsArray = body.urls;
+
+    if (urlsArray.length) {
+      for (let url of urlsArray) {
+        const formattedUrl = url.replace(requestUrl.origin + '/', '');
+
+        const file = Bun.file(path.join(storePath, formattedUrl));
+        const isExists = await file.exists();
```
Comment (634750802) [inline src/handlers/del.ts]: It's better to check the `._vercel_mock_meta_` file.

File: src/handlers/del.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  },
+
+  async handle(requestUrl, request) {
+    const body = await request.json();
+
+    const urlsArray = body.urls;
+
+    if (urlsArray.length) {
+      for (let url of urlsArray) {
+        const formattedUrl = url.replace(requestUrl.origin + '/', '');
```
Comment (634750802) [inline src/handlers/del.ts]:
```ts
const urlObject = new URL(url);
const pathname = urlObject.pathname; // your formattedUrl with a prefix '/'
// Optional: verify the origin
// assert urlObject.origin === requestUrl.origin
```

**Review by 634750802 — CHANGES_REQUESTED** [review]
Thanks for your contribute!

File: src/handlers/del.ts (hunk 2 above)
Comment (sarmat86) [reply src/handlers/del.ts]: I agree, added.

File: src/handlers/del.ts (hunk 3 above)
Comment (sarmat86) [reply src/handlers/del.ts]: Del function from Vercel blob can take as an argument whole url or pathname. If you try to make a URL object form pathname you'll get an error. 
But anyway, I've refactored code a little, seems more readable now.

File: src/handlers/del.ts (hunk 1 above)
Comment (sarmat86) [reply src/handlers/del.ts]: console.log deleted

**Pull request merged**